### PR TITLE
kd: add interactive mode for "credential create"

### PR DESCRIPTION
**Pull request by rjeczalik**
depends on https://github.com/koding/koding/pull/9666

### CODE REVIEW

File: go/src/koding/kites/kloud/stack/provider/describe_test.go [hunk 1]
```diff
@@ -3,18 +3,21 @@ package provider_test
...
 )
 
 type Foo struct {
-	File []byte `kloud:",secret"`
-	N    int    `kloud:",readOnly"`
-	S    string
-	M    map[string]interface{} `kloud:"someMap"`
-	E    FooEnum                `json:"enums" kloud:"someEnums"`
+	File          []byte `kloud:",secret"`
+	N             int    `kloud:",readOnly"`
```
Comment (cihangir) [inline go/src/koding/kites/kloud/stack/provider/describe_test.go]: Acrostic?

**Review by cihangir — APPROVED** [review]

File: go/src/koding/kites/kloud/stack/provider/describe_test.go (hunk 1 above)
Comment (rjeczalik) [inline go/src/koding/kites/kloud/stack/provider/describe_test.go]: Nope, just one-letter names due to laziness - `n` is usually int, `s` is usually string etc.

Comment (cihangir) [inline go/src/koding/kites/kloud/stack/provider/describe_test.go]: j/k

**Review by ppknap — APPROVED** [review]

File: go/src/koding/kites/kloud/stack/provider/describe.go [hunk 2]
```diff
@@ -113,3 +122,30 @@ func mustDescribe(v interface{}) []stack.Value {
 	}
 	return desc
 }
+
+var titler = strings.NewReplacer("_", " ")
+
+func title(s string) string {
```
Comment (rjeczalik) [inline go/src/koding/kites/kloud/stack/provider/describe.go]: Ping. Merging for now.

I find it harder to reason as in your version there's more "magic" behind regexp functors.

However if you find a use-case that existing impl is not enough, feel encouraged to send PR with your version.

**Pull request merged**